app/server/lib/AttachmentFileManager.ts

```diff
@@ -141,7 +141,12 @@ export class AttachmentFileManager extends EventEmitter {
 
 
   public async getFileData(fileIdent: string): Promise<Buffer> {
-    return (await this._getFile(fileIdent)).data;
+    const file = await this.getFile(fileIdent);
+    return (await loadAttachmentFileIntoMemory(file)).contents;
+  }
+
+  public async getFile(fileIdent: string): Promise<AttachmentFile> {
```

Comment:
Having a _getFile() and a getFile() method that do something different is confusing. Rename _getFile() to _getFileInfo?